UTKcataloging/mods_to_rdf
# Text streamlining and contributors

**JIRA Issue**: [DIGITAL-937](https://jirautk.atlassian.net/browse/DIGITAL-937)

What Does this Do?
==================

This attempts to make the "Use Case" and "Justification" sections more consistent in content and length. To do this, I focused on answering particular questions for each section:

**Use Case**
- How are the values currently being used / displayed / shared in MODS?

**Justification**
- Why are we (or why are we not) keeping the values defined in the particular category?
- Do they need to be separate from other values (aka, what justifies using a distinct property for the group of values)?
- Why did we select a particular property?

In addition, the following changes were made:
- added everyone in the group as a contributor
- a section for partNumber was added to the title section
- note[@type="provenance"] was removed from the possible Xpaths in the general note section as all values associated with this Xpath were deleted in [DIGITAL-780](https://jirautk.atlassian.net/browse/DIGITAL-780)
- captions for code blocks were removed from the name section for consistency's sake (I think these actually improve the code blocks, but we'd need to agree to do this across the whole document)
- typo corrections, etc.

How Should This Be Tested?
==========================

Please particularly review the larger changes I noted above. In addition, focus on reviewing the sections you contributed to see if I have inadvertently changed your meaning / intentions.

Additional Notes
================

Add anything else you want or that would be useful.
